### PR TITLE
Add aflturbo

### PR DESCRIPTION
**Pull request by jafarsarif10**
Changes:

1. Added guide for building and using AFLturbo for fuzzing.
2. script to run AFLTurbo for each fuzzing test case.

### CODE REVIEW

File: unit_test/fuzzing/fuzzing_AFLTurbo.sh [hunk 1]
```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+# Before run this script, please Download and install AFL. Unzip and follow docs\QuickStartGuide.txt. Build it with make.
```
Comment (mahendrakumarprajapat) [inline unit_test/fuzzing/fuzzing_AFLTurbo.sh:3]: rename AFL to AFLTurbo

File: unit_test/fuzzing/fuzzing_AFLTurbo.sh [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    echo "<CRYPTO> means selected Crypto library: mbedtls or openssl"
+    echo "<GCOV> means enable Code Coverage or not: ON or OFF"
+    echo "<duration> means the duration of every program keep fuzzing: NUMBER seconds"
+    exit
+fi
+
+echo "<duration> parameter is $3"
+export duration=$3
+
+echo "start fuzzing in Linux with AFL"
```
Comment (mahendrakumarprajapat) [inline unit_test/fuzzing/fuzzing_AFLTurbo.sh:39]: rename AFL to AFLTurbo

**Comment by jyao1** [comment]
please fix CI check

**Comment by jyao1** [comment]
@jafarsarif10, please add "Signed-off-by: " tag to your commit message to resolve CI failure

**Comment by mahendrakumarprajapat** [comment]
> @jafarsarif10, please add "Signed-off-by: " tag to your commit message to resolve CI failure

@jyao1 : Yes, Jafar will do it by today.

**Comment by jafarsarif10** [comment]
Will generate new PR with the required changes. Hence closing this PR as it is not required.

**Pull request closed without merging**